src/main/java/pl/agh/edu/master_diet/service/ActivitySearchService.java

```diff
@@ -0,0 +1,36 @@
+package pl.agh.edu.master_diet.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pl.agh.edu.master_diet.core.model.database.Activity;
+import pl.agh.edu.master_diet.core.model.rest.browser.activity.ActivitySearchResponse;
+import pl.agh.edu.master_diet.core.model.shared.SearchResult;
+import pl.agh.edu.master_diet.repository.ActivityRepository;
+import pl.agh.edu.master_diet.service.converter.ConversionService;
+
+import java.util.stream.Collectors;
+
+@Service
+public class ActivitySearchService extends SearchService<Activity> {
+
+
```

Comment:
This newline is redundant I guess :smile:

Reply:
Deleted.